recipe/meta.yaml

```diff
@@ -99,8 +95,8 @@ outputs:
     script: install_llvm.sh  # [not win]
     build:
       activate_in_script: true
-      run_exports:                                                    # [not win]
-        - {{ pin_subpackage("libllvm" + major_ver, max_pin="x.x") }}  # [not win]
+      run_exports:                                                    # [unix]
+        - {{ pin_subpackage("libllvm" ~ major_ver, max_pin="x.x") }}  # [unix]
```

Comment:
This issue is about windows, so `not win` is appropriate here.

Reply:
I prefer the non-inverted `unix` over `not win` and use this synonymously.

Reply:
Let's keep `not win` here. I prefer it because it makes things clear that this works for everywhere except windows.
`unix` and `not win` are not synonyms in conda-build due to the future `wasi-*` platform.